src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -623,6 +630,11 @@ public LicensedVideoSwapPageObject openLicensedVideoSwap(String wikiURL) {
     return new LicensedVideoSwapPageObject(driver);
   }
 
+  public TemplatePageObject openTemplatePage(String wikiURL, String template) {
+    getUrl(wikiURL + URLsContent.TEMPLATE_PAGE + template);
+    return new TemplatePageObject(driver);
+  }
+
   @Deprecated
   public void verifyAvatarPresent() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)